temba/contacts/views.py

```diff
@@ -1464,71 +1464,27 @@ def get_gear_links(self):
             return links
 
         def get_context_data(self, *args, **kwargs):
-            context = BulkActionMixin.get_context_data(self, *args, **kwargs)
+            context = super().get_context_data(*args, **kwargs)
             org = self.request.user.get_org()
             group = self.derive_group()
             view_url = reverse("contacts.contact_invite_participants")
 
+            context["folders"] = context["folders"][:1]
```

Comment:
Can we have `context["folders"]` like here https://github.com/greatnonprofits-nfp/rapidpro/blob/dev-2.0/temba/contacts/views.py#L1810?

Reply:
Updated